index.html

```diff
@@ -1,413 +1,308 @@
-      sizes="180x180"
-      href="/images/apple-touch-icon.png"
-    />
-    <link
-      rel="icon"
-      type="image/png"
-      sizes="32x32"
-      href="/images/favicon-32x32.png"
-    />
-    <link
-      rel="icon"
-      type="image/png"
-      sizes="16x16"
-      href="/images/favicon-16x16.png"
-    />
-    <link rel="manifest" href="site.webmanifest" />
-    <link
-      rel="mask-icon"
-      href="/images/safari-pinned-tab.svg"
-      color="#5bbad5"
-    />
-    <meta name="msapplication-TileColor" content="#da532c" />
-    <meta name="theme-color" content="#ffffff" />
-    <link
-      href="https://fonts.googleapis.com/css?family=Lato|Oswald"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="styles.css" />
-    <link
-      rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
-    />
-    <title>NEW DEVS ON THE BLOCK</title>
-  </head>
-  <body>
-    <header>
-      <img
-        src="images/logo.png"
-        class="logo"
-        alt="kickass new devs on the block logo"
-      />
-    </header>
-    <main>
+    that it’s all completely free of charge." />
+  <meta name="keywords"
+    content="New Devs on the Block, Vienna, Workshop, HTML, CSS, Martin Splitt, Eva Lettner, Cassie Evans, Nico König, Ramón Huidobro" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png" />
+  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png" />
+  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png" />
+  <link rel="manifest" href="site.webmanifest" />
+  <link rel="mask-icon" href="/images/safari-pinned-tab.svg" color="#5bbad5" />
+  <meta name="msapplication-TileColor" content="#da532c" />
+  <meta name="theme-color" content="#ffffff" />
+  <link href="https://fonts.googleapis.com/css?family=Lato|Oswald" rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" />
+  <link rel="stylesheet" href="styles.css" />
+  <title>NEW DEVS ON THE BLOCK</title>
+</head>
+
+<body>
+  <header id="top">
+    <nav>
+      <a href="#schedule">Schedule</a>
+      <a href="#workshops">Workshops & Register</a>
+      <a href="#coaches">Coaches</a>
+      <a href="#partners">Partners</a>
+    </nav>
+    <img src="images/logo.png" class="logo" alt="kickass new devs on the block logo" />
+  </header>
+  <main>
+    <article id="about">
+      <h2>About us</h2>
+      <p>
+        Have you always wanted to build your first own website, but somehow, you didn't know how to get started? Maybe
+        because you thought you were not that techie after all? Then these two days are precisely for you! We will take
+        the
+        first few steps down the web development road together.
```

Comment:
```suggestion
Have you always wanted to build your first own website, but didn't know where to start? Or maybe you thought you weren't technical enough? Then these two days are for you! We'll take the first few steps down the web development road together.
```

I would also consider adding back something like `The best part? It’s completely free!` because then that'll help ease anyone who is worried about the price.